sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DefaultDocumentQueryExecutionContext.java

```diff
@@ -185,7 +185,8 @@ protected PartitionKeyInternal getPartitionKeyInternal() {
                                                     tFeedResponse.getRequestCharge(),
                                                     this.fetchExecutionRangeAccumulator.getExecutionRanges(),
                                                     Arrays.asList(schedulingTimeSpanMap)),
-                                            tFeedResponse.getActivityId());
+                                            tFeedResponse.getActivityId(),
+                                        tFeedResponse.getResponseHeaders().getOrDefault(HttpConstants.HttpHeaders.INDEX_UTILIZATION, null));
```

Comment:
why we have null in last parameter , also i am not seeing changes  for ClientSideMetrics ctor